tests/unit/test_backends.py

```diff
@@ -93,26 +93,10 @@ def convertInfoValue(column, value):
     return ret
 
 
-class TestWormtableBackend(unittest.TestCase):
+class TestAbstractBackend(unittest.TestCase):
 
     def setUp(self):
-        global _wormtableTestFixture
-        self._dataDir = _wormtableTestFixture.dataDir
-        self._variantsDir = _wormtableTestFixture.variantsDir
-        self._tables = {}
-        self._chromIndexes = {}
-        self._chromPosIndexes = {}
-        for relativePath in os.listdir(self._variantsDir):
-            table = wt.open_table(
-                os.path.join(self._variantsDir, relativePath))
-            self._tables[relativePath] = table
-            self._chromIndexes[relativePath] = table.open_index("CHROM")
-            self._chromPosIndexes[relativePath] = table.open_index("CHROM+POS")
-        self._backend = backend.FileSystemBackend(self._dataDir)
-
-    def tearDown(self):
-        for table in self._tables.values():
-            table.close()
+        self._backend = backend.AbstractBackend()
```

Comment:
I don't think there's much point in instantiating an AbstractBackend here, as this will get over-written in the subclass anyway. We can just get rid of `setUp` here.